common/Reducers.jsm

```diff
@@ -386,6 +390,18 @@ function Pocket(prevState = INITIAL_STATE.Pocket, action) {
   switch (action.type) {
     case at.POCKET_WAITING_FOR_SPOC:
       return {...prevState, waitingForSpoc: action.data};
+    case at.POCKET_LOGGED_IN:
+      return {...prevState, isUserLoggedIn: !!action.data};
+    case at.POCKET_CTA:
+      return {
+        ...prevState,
+        pocketCta: {
+          ctaButton: action.data.cta_button,
```

Comment:
why use camel case v.s. underscores differently?

Reply:
Initially, was just trying to match the case in the object keys on browser.newtabpage.activity-stream.feeds.section.topstories.options.

Once I started to build it out I decided I wanted to put all the values onto the state object `ctaButton` anyway, and at this point it made sense to keep everything in camel case.

So I guess just trying to match case without going too far out of my way.

Reply:
👍